TRUSTED-COMMITTERS.md

```diff
@@ -0,0 +1,37 @@
+# Trusted Committers
+
+Trusted committers (TCs) are those members of our working group who have elevated rights and direct write access to this repository. *TCs act as stewards of the working group and community. They aim to make consensus-based decisions in the best interest of the working group.* They also act as the guardians of this repository: TCs react to, referee, and give feedback about incoming contributions.
+
+## Current Trusted Committers
+
+* @lenucksi
+* @nyeates
+* @gruetter
+* @NewMexicoKid
+* @cewilliams
+
+## Process for Adding new Trusted Committers
+
+We work based on trust: Our goal is to add most people who contributed a sizeable change - quick and early.
+
+We follow this process (adapted from [here](https://tech.europace.de/voting-in-new-trusted-committers/)):
+
+1. Any trusted committer (TC) can step forward and nominate a new TC in the private Slack channel #innersource-patterns-tcs. The TC should provide the following information:
+   * Name of the candidate
+   * Reason for candidate
+   * Github handle of the candidate
+   * Slack handle of the candidate
+1. Every TC can raise concerns or second the nomination in the #innersource-patterns-tcs channel.
+1. If none of the existing TCs disagrees with the nomination within 72h, [lazy consensus](https://tech.europace.de/lazy-consensus-vs-explicit-voting/) is reached: The nomination is accepted.
+1. The TC who nominated the candidate informs her/him in private about the nomination and its acceptance. The candidate can decide on whether to accept or reject the offer.
+1. If the candidate accepts the offer, the TC who nominated the candidate, makes sure:
+   * New TC is added to this file
+   * New TC receives write access to this repository
+   * New TC is added to the #innersource-patterns-tcs channel
+   * New TC is praised on the general InnerSource Commons [mailing list](mailto:innersource-commons@googlegroups.com)
```

Comment:
```suggestion
   * New TC is praised in the [#innersource-patterns](https://app.slack.com/client/T04PXKRM0/C2EFRTS6A) channel.
```

I don't think that the entire commons needs to be notified - just the people that are interested in the InnerSource Patterns working group.

Reply:
I agree with the proposed change (and already committed it to the PR branch). 

For documentation purpose: Our idea was to give praise to a wide audience, also to recognize (and motivate) new committers. I agree though, for the beginning, it will be likely be sufficient to announce it in the patterns channel.